haas/cli.py

```diff
@@ -400,153 +439,183 @@ def switch_register(switch, subtype, *args):
-            sys.stderr.write('<hostname> <username> <password> <dummy_vlan_no>\n')
+            sys.stderr.write('ERROR: subtype ' + subtype +
+                             ' requires exactly 4 arguments\n')
+            sys.stderr.write(
+                '<hostname> <username> <password> <dummy_vlan_no>\n')
             return
     elif subtype == "mock":
         if len(args) == 3:
-            switchinfo = { "type": switch_api+subtype, "hostname": args[0],
-                        "username": args[1], "password": args[2] }
+            switchinfo = {"type": switch_api + subtype, "hostname": args[0],
+                          "username": args[1], "password": args[2]}
         else:
-            sys.stderr.write('ERROR: subtype '+subtype+' requires exactly 3 arguments\n')
+            sys.stderr.write('ERROR: subtype ' + subtype +
+                             ' requires exactly 3 arguments\n')
             sys.stderr.write('<hostname> <username> <password>\n')
             return
     elif subtype == "powerconnect55xx":
         if len(args) == 3:
-            switchinfo = { "type": switch_api+subtype, "hostname": args[0],
-                        "username": args[1], "password": args[2] }
+            switchinfo = {"type": switch_api + subtype, "hostname": args[0],
+                          "username": args[1], "password": args[2]}
         else:
-            sys.stderr.write('ERROR: subtype '+subtype+' requires exactly 3 arguments\n')
+            sys.stderr.write('ERROR: subtype ' + subtype +
+                             ' requires exactly 3 arguments\n')
             sys.stderr.write('<hostname> <username> <password>\n')
             return
     elif subtype == "brocade":
         if len(args) == 4:
-            switchinfo = { "type": switch_api+subtype, "hostname": args[0],
-                           "username": args[1], "password": args[2],
-                           "interface_type": args[3] }
+            switchinfo = {"type": switch_api + subtype, "hostname": args[0],
+                          "username": args[1], "password": args[2],
+                          "interface_type": args[3]}
         else:
-            sys.stderr.write('ERROR: subtype '+ subtype+' requires exactly 4 arguments\n')
-            sys.stderr.write('<hostname> <username> <password> <interface_type>\n')
-            sys.stderr.write('NOTE: interface_type refers to the speed of the switchports\n')
-            sys.stderr.write('ex. TenGigabitEthernet, FortyGigabitEthernet, etc.\n')
+            sys.stderr.write('ERROR: subtype ' + subtype +
+                             ' requires exactly 4 arguments\n')
+            sys.stderr.write(
+                '<hostname> <username> <password> <interface_type>\n')
+            sys.stderr.write(
+                'NOTE: interface_type refers to the speed of the switchports\n')
+            sys.stderr.write(
+                'ex. TenGigabitEthernet, FortyGigabitEthernet, etc.\n')
```

Comment:
That is actually what autopep8 or yapf would do by default. I would be more comfortable if we would do sth like this

```
if not kwargs:
    sys.stderr.write(_("Role not added, incorrect set of arguments "
                       "provided. See openstack --help for more "
                       "details\n"))
```

@matsuuran Having one or more ``sys.stderr.write`` is just your call.